pragma/tests/fetchers_test.py

```diff
@@ -378,13 +464,26 @@ def test_future_fetcher_sync_success(future_fetcher_config, mock_future_data):
             SAMPLE_FUTURE_ASSETS, PUBLISHER_NAME
         )
 
+        other_mock_fns = future_fetcher_config.get("other_mock_fns", {})
```

Comment:
could that be made a fixture ?

Reply:
Yep, I think that would make it nicer as well.

Reply:
Yes, I think that's a good idea!